README.md

```diff
@@ -0,0 +1,23 @@
+# OCF Status Page
+
+This status page is currently available at http://dev-status.ocf.io.
+
+## Running
+
+To run a dev version and make sure your changes look good:
+
+```sh
+make dev
+```
+
+To deploy your changes (this should happen *before* (!) you commit
+your changes):
+
+```sh
+make deploy
+```
```

Comment:
While I think these instructions should still exist, to be used in the event that Jenkins is unavailable, I think we should still use Jenkins for deployment in the normal case.